supplementary_style_guide/glossary_terms_conventions/general_conventions/h.adoc

```diff
@@ -22,7 +22,7 @@
 
 [[hashbang]]
 ==== image:images/no.png[no] hashbang (noun)
-*Description*: Do not use _hashbang_ when referring to a computer language construct that controls which interpreter parses and interprets instructions in a computer program. Instead, on the first mention, include both "interpreter directive" and "shebang", for example, "Interpreter directives, also known as shebangs ...".
+*Description*: Do not use _hashbang_ when referring to a computer language construct that controls which interpreter parses and interprets instructions in a computer program. Write "interpreter directives, also known as shebangs ..." on first use and "interpreter directives" after that.
```

Comment:
Same comment as below, I think they can use "shebang" after that. Or at least, Miriam should clarify what we want to say here. Or we could leave it off and just say the part about what to say on first use.

Reply:
Thanks. Updated.